Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.0
+// swift-tools-version:5.8
```

Comment:
just wanted to propose this change, thank you!

Reply:
Thanks, but going in favor of https://github.com/devicekit/DeviceKit/pull/381 for this specific change.